src/controllers/enterpriseController.test.ts

```
@@ -93,17 +95,129 @@ describe("enterpriseController", () => {
     );
   });
 
-  it("keeps the treasury endpoint stub intact", async () => {
+  function mockGetTreasury() {
+    return (jest.requireMock("../services/treasury/TreasuryService") as any).getEnterpriseTreasury as jest.Mock<any>;
```

ESLint

[error] Insert `⏎······`

(prettier/prettier)